### PR TITLE
Added 3 libs to conda and 1 to pip

### PR DESCRIPTION
**Pull request by xalron**
Added 4 libraries to be used in the challenge.

### CODE REVIEW

File: runtime/environment-cpu.yml [hunk 1]
```diff
@@ -36,5 +39,6 @@ dependencies:
       - gower==0.1.2
       - interpret==0.3.0
       - git+https://github.com/ibarrond/Pyfhel.git@0dee7a77469627c4de4f2dca97cdc68bf006a77d
+      - py-polar-codes==1.2.2
```
Comment (jayqi) [inline runtime/environment-cpu.yml:42]: Can you please explain the purpose of this package? My understanding is that polar codes are a type of error-correcting code. Controlling errors over communication channels is not the expected abstraction layer for this challenge.

File: runtime/environment-cpu.yml [hunk 2]
```diff
@@ -7,11 +7,14 @@ dependencies:
...
   - imbalanced-learn=0.10.0
   - jax=0.3.20
+  - fuzzywuzzy=0.18.0
+  - gensim=3.8.3
   - keras=2.10.0
   - lightgbm=3.1.1
   - loguru=0.6.0
   - mkl=2022.1.0
   - networkx=2.8.8
+  - nltk=3.8.1
```
Comment (jayqi) [inline runtime/environment-cpu.yml:17]: Can you explain how you plan to use some of these natural language processing libraries? There is very limited text data in this challenge. Are you using this to process text, or you planning to represent other types of data as if they were text and using NLP algorithms?

File: runtime/environment-cpu.yml (hunk 1 above)
Comment (xalron) [reply runtime/environment-cpu.yml:42]: The Polar codes will not be used as per their error control capability, but with Regards to converting data to a noise-like distribution to add additional layer of security.

File: runtime/environment-cpu.yml (hunk 2 above)
Comment (xalron) [reply runtime/environment-cpu.yml:17]: We have obtained some success in generating a corpus (without providing external data) in our projects and we'll be using it in context to improve our predictions for this challenge as well, while exploring to see if  NLP'esque approaches are viable for privacy protecting models.

**Comment by jayqi** [comment]
@xalron Thanks for the follow-up explanations. I've incorporated these changes as part of #33, which has been merged in. Closing this PR as done.

**Pull request closed without merging**